x/supplier/types/errors.go

```diff
@@ -15,4 +15,6 @@ var (
 	ErrSupplierInvalidServiceConfig      = sdkerrors.Register(ModuleName, 5, "invalid service config")
 	ErrSupplierInvalidSessionStartHeight = sdkerrors.Register(ModuleName, 6, "invalid session start height")
 	ErrSupplierInvalidSessionId          = sdkerrors.Register(ModuleName, 7, "invalid session ID")
+	ErrSupplierInvalidService            = sdkerrors.Register(ModuleName, 8, "invalid service")
```

Comment:
Is there not already an error like this in the service package and/or would this make more sense to live there instead?

Reply:
@bryanchriswhite To be completely honest, I haven't thought too deeply about this.

My argument would be that we Register the error with this (supplier's) module name, so it would perculate up to the caller and help identify the source.

Do you want to leave a TODO for this or a preference on addressing it now?

Reply:
The change you made in https://github.com/pokt-network/poktroll/pull/123#discussion_r1385613906, restores consistency. Wdyt about using the more verbose phrasing you applied in #123 here as well?:
```suggestion
	ErrSupplierInvalidService            = sdkerrors.Register(ModuleName, 8, "invalid service in supplier")
```

Reply:
Absolutely